config/crds/apm.k8s.elastic.co_apmservers.yaml

```diff
@@ -34,342 +34,6 @@ spec:
-                secure setting to be injected. You can use the `entries` and `key`
-                fields to consider only a subset of the secret entries and the `path`
-                field to change the target path of a secret entry key. The secret
-                must exist in the same namespace as the APM resource.
-              items:
-                properties:
-                  entries:
-                    description: If unspecified, each key-value pair in the Data field
-                      of the referenced Secret will be projected into the volume as
-                      a file whose name is the key and content is the value. If specified,
-                      the listed keys will be projected into the specified paths,
-                      and unlisted keys will not be present.
-                    items:
-                      description: Maps a string key to a path within a volume.
-                      properties:
-                        key:
-                          description: The key to project.
-                          type: string
-                        path:
-                          description: The relative path of the file to map the key
-                            to. May not be an absolute path. May not contain the path
-                            element '..'. May not start with the string '..'.
-                          type: string
-                      required:
-                      - key
-                      type: object
-                    type: array
-                  secretName:
-                    description: 'Name of the secret in the pod''s namespace to use.
-                      More info: https://kubernetes.io/docs/concepts/storage/volumes#secret'
-                    type: string
-                required:
-                - secretName
-                type: object
-              type: array
-            version:
-              description: Version represents the version of the APM Server
-              type: string
-          type: object
-        status:
-          description: ApmServerStatus defines the observed state of ApmServer
-          properties:
-            associationStatus:
-              description: Association is the status of any auto-linking to Elasticsearch
-                clusters.
-              type: string
-            availableNodes:
-              type: integer
-            health:
-              description: ApmServerHealth expresses the status of the Apm Server
-                instances.
-              type: string
-            secretTokenSecret:
-              description: SecretTokenSecretName is the name of the Secret that contains
-                the secret token
-              type: string
-            service:
-              description: ExternalService is the name of the service the agents should
-                connect to.
-              type: string
-          type: object
-      type: object
```

Comment:
Now that we introduce a difference between versions, the schema is gone for APM/Kibana too :-(

Reply:
Well, at least that's consistent 😄